src/entity/user/User.ts

```diff
@@ -22,6 +22,12 @@ export type UserFlags = Partial<{
   trustScore: number;
 }>;
 
+export type UserSubscriptionFlags = Partial<{
+  monthly: boolean;
+  yearly: boolean;
```

Comment:
Instead of two booleans should we consider a string?

Reply:
Why the string? what does it contain? the subscription_id?

Reply:
monthly and yearly as two separate variables is confusing. it's basically a billing cycle. so we can have a variable named cycle as enum